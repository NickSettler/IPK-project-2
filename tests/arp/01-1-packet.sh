./ipk-sniffer -i en0 --arp