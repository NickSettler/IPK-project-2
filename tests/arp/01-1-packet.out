timestamp: 2023-04-17T14:24:29.076+02:00
src MAC: 98:77:e7:50:60:5e
dst MAC: b0:be:83:4a:c4:b0
frame length: 52 bytes
src IP: 192.168.1.138
dst IP: 192.168.1.17

0x0000: b0 be 83 4a c4 b0 98 77 e7 50 60 5e 08 06 00 01  ...J...w.P`^....
0x0010: 08 00 06 04 00 01 98 77 e7 50 60 5e c0 a8 01 8a  .......w.P`^....
0x0020: 00 00 00 00 00 00 c0 a8 01 11 00 00 00 00 00 00  ................
0x0030: 00 00 00 00                                      ....

